code/modules/admin/antagifiers.dm

```diff
@@ -225,3 +225,13 @@
 			var/token = new /obj/item/requisition_token/syndicate/vr(get_turf(M))
 			M.put_in_hand_or_eject(token) // try to eject it into the users hand, if we can
 			boutput(M, "<span class='combat'>Redeem your freshly mined syndicoin in the nearby weapon vendor.</span>")
+
+	arcfiend
+		name = "ARCF13ND.EXE"
+		desc = "Turns you into an arcfiend, using a bit of the processing power of vr."
+		icon = 'icons/obj/power.dmi'
+		icon_state = "apc0"
+
+		makeAntag(mob/living/carbon/human/M as mob)
+			M.make_arcfiend(1)
```

Comment:
whats the 1 for

Reply:
Not really sure myself. Some antagifiers had it, others didn't. The vr ones did, so I figured it needed it. I'll check if I can remove it once I get back home.